tests/lax_numpy_test.py

```diff
@@ -4671,15 +4671,21 @@ def testMgrid(self):
                         rtol=rtol)
 
   def testOgrid(self):
-    assertAllEqual = partial(self.assertAllClose, atol=0, rtol=0)
-    assertAllEqual(np.ogrid[:5], jnp.ogrid[:5])
-    assertAllEqual(np.ogrid[:5,], jnp.ogrid[:5,])
-    assertAllEqual(np.ogrid[:5], jax.jit(lambda: jnp.ogrid[:5])())
-    assertAllEqual(np.ogrid[0:5, 1:3], jnp.ogrid[0:5, 1:3])
-    assertAllEqual(np.ogrid[1:7:2], jnp.ogrid[1:7:2])
-    assertAllEqual(np.ogrid[:5, :9, :11], jnp.ogrid[:5, :9, :11])
+    def assertListOfArraysEqual(xs, ys):
+      self.assertEqual(len(xs), len(ys))
+      for x, y in zip(xs, ys):
+        self.assertArraysEqual(x, y)
+
+    self.assertArraysEqual(np.ogrid[:5], jnp.ogrid[:5])
+    self.assertArraysEqual(np.ogrid[:5,], jnp.ogrid[:5,])
```

Comment:
This is technically a list of arrays, isn't it?

Reply:
Indeed.
Updated PR!